### PR TITLE
Bring MostRead out of alpha

### LINKED ISSUE #2994: Make MostRead a non-alpha component
**Is your feature request related to a problem? Please describe.**
Once all accessibility and UX issues have been addressed, move MostRead out of alpha.

**Describe the solution you'd like**
~Just do it~ Confirm and finish all accessibility requirements, move MostRead out of alpha and begin https://github.com/bbc/simorgh/issues/4986

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by hotinglok**
Resolves #2994 

**Overall change:** 
Makes MostRead a non-alpha component ready to be pulled into simorgh

**Code changes:**
- Removed alpha tag from various files such as `changelog.md` and `package.json`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-most-read/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.0.0 | [PR#xxxx](https://github.com/bbc/psammead/pull/xxxx) Bring MostRead out of alpha |
```
Comment (OlgaLyubin) [inline packages/components/psammead-most-read/CHANGELOG.md]:
```suggestion
| 1.0.0 | [PR#3021](https://github.com/bbc/psammead/pull/3021) Bring MostRead out of alpha |
```

**Review by OlgaLyubin — COMMENTED** [review]
In "Accessibility notes" section of README you have "Currently this component is in alpha. This is because it has not yet been tested with various assistive technologies. After it has had an accessibility swarm, this will be published under a standard version." Can you update this please?

**Review by OlgaLyubin — APPROVED** [review]
Looks good 👍

**Review by DenisHdz — APPROVED** [review]
LGTM

File: packages/components/psammead-most-read/README.md [hunk 2]
```diff
@@ -1,9 +1,5 @@
  <!-- prettier-ignore -->
```
Comment (Bopchy) [inline packages/components/psammead-most-read/README.md]:
```suggestion
 
```

Comment (Bopchy) [reply packages/components/psammead-most-read/README.md]: Should we remove this? I don't think it's needed

**Review by Bopchy — APPROVED** [review]

**Pull request merged**